source/codegen/metadata/nirfmxspecan_restricted/config.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+config = {
+    "api_version": "21.0.0",
```

Comment:
I copied this from the nirfmxspecan/config.py file to keep them in sync.

Reply:
@astarche or @reckenro, do either of you have guidance on this `api_version` and the `latest_runtime_version_tested_against` question below?

Reply:
Yeah, so I think this is a remnant of us starting from nimi-python metadata. We copied it over as a starting point but never cleaned up what we weren't using. nimi-python uses api_version, module_version, and latest_runtime_version_tested_against fields for different parts of their build but we haven't yet.

As for guidance, for this PR I think it's fine to match nirfmxspecan's values for these since they're not used. I think we should do a metadata cleanup as a whole for grpc-device to decide what values we can clear out. We have some metadata related features coming up on our squad so I think I'll bring this back up when we get to those.